tests/acceptance/recipes/new-test.js

```diff
@@ -0,0 +1,27 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'recipe-card-19/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | recipes/new');
+
+test('User can visit a recipe page and see a recipe', function(assert) {
+  visit('/');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/');
+  });
+});
+
+test('User can see the serving size of the recipe', function(assert) {
+  let number = '.num-servings-input';
+  visit('/');
+
+  fillIn(number);
+
+  // click('.adjust-recipe');
+
+  andThen(function() {
+
+    assert.equal(number = 8);
```

Comment:
If you are asserting that `number` is equal to `8` then you would use `assert.equal(number, 8)` here is the docs from qUnit: http://api.qunitjs.com/equal/